guides/development/hack_on_the_trento.md

```diff
@@ -1,82 +1,111 @@
+2. [Docker](https://docs.docker.com/get-docker/) - a platform for developing, shipping, and running applications in containers
+3. [Docker Compose](https://docs.docker.com/compose/install/) - a tool for defining and running multi-container Docker applications.
 
-1. [Elixir](https://elixir-lang.org/)
-2. [Erlang OTP](https://www.erlang.org/)
-3. [Docker](https://docs.docker.com/get-docker/)
-4. [Docker Compose](https://docs.docker.com/compose/install/)
+**Note** that you also need to have Git installed on your computer to clone the Trento repository.
 
 ## Development environment
 
-A `docker-compose` development environment is provided.
+The Trento project provides a docker-compose development environment that you can use to start a Postgres database and a Grafana instance for storage and monitoring. To start the development environment, navigate to the root directory of the Trento project and run the following command:
 
 ```
 docker-compose up -d
 ```
 
-It will start a **postgres** database and a **grafana** instance, for storage and monitoring.
-## Install dependencies
+This starts the Postgres and Grafana containers in detached mode, so you can continue to use your terminal for other commands.
+
+It starts a **postgres** database and a **grafana** instance, for storage and monitoring.
+
+## Setup Trento
+
+Before starting Trento Web, some initial setup tasks are required. This can be achieved by running the following command:
 
 ```
-mix deps.get
+mix setup
 ```
 
-## Setup trento
+This command performs necessary tasks such as installing dependencies, creating the database schema and running migrations.
+
+### Hint about Project setup
+
+Gain a deeper understanding of how Trento Web is configured, reading the [mix.exs](https://github.com/trento-project/web/blob/main/mix.exs) file located in the root directory of the project. This file contains information on dependencies, configuration settings, and tasks that can be run using the Mix build tool, providing a complete picture of the project's setup.
+
+## Connecting Trento Web to [Wanda](https://github.com/trento-project/wanda)
+
+Wanda is a service that is responsible to orchestrate Checks for the Trento Web application. By default, Wanda can be accessed on port 4001. To connect Trento Web to Wanda, you need to create a .env file in the root path of Trento Web.
+
+Create the file by running the following command in your terminal:
 
 ```
-mix setup
+echo "WANDA_URL=http://localhost:4001" > assets/.env
 ```
 
+**Guide** how to hack on [Wanda](https://github.com/trento-project/wanda/blob/main/guides/development/hack_on_wanda.md).
+
+### Hint about the default port of Wanda
+
+If the Wanda service is running on a different port, you need to change 4001 in the .env file to the correct port number.
+
 ## Install the JavaScript frontend packages
 
+The Trento application also requires some JavaScript packages to be installed for the frontend. To install these packages, run the following command:
```

Comment:
I don't think we need to start repeating the `Trento application` all the time. `The application` should be enough I would say